app/components/meeting/edit-meeting-modal.js

```diff
@@ -231,26 +231,44 @@ export default class MeetingEditMeetingComponent extends Component {
     this.args.meeting.numberRepresentation = this.numberRepresentation;
     this.args.meeting.mainMeeting = this.selectedMainMeeting;
 
-    if (this.enableDigitalAgenda) {
```

Comment:
**question**: is moving this whole block outside of the feature flag intentional?